frontend/src/components/pages/Scheduling/SelectDateTime.tsx

```diff
@@ -145,6 +154,13 @@ const SelectDateTime = ({
     setIcons(getIconsPerTimeSlot(selectedDayPart, selectedDate));
   };
 
+  const checkSubmit = (hasRecurringDonationEndDate: boolean) => {
+    const reccurring = hasRecurringDonationEndDate ? frequency && recurringDonationEndDate : true;
```

Comment:
i think this is where your bug is occurring? maybe? i console.log(recurring) and it is giving me an undefined which will then affect your valuesFilled